web/views/flow_step.erb

```diff
@@ -31,6 +31,7 @@
   <% end %>
   <%= show_action_data("Input:", action.input) %>
   <%= show_action_data("Output:", action.output) %>
+  <%= show_action_data("Chunked output:", action.stored_output_chunks) %>
```

Comment:
Should we display this to the user, shouldn't we join the chunks when showing?

Reply:
Can we do that? I mean we provide an option to store data in an append only fashion, but we don't really care what the data is. Some of it may not be join-able

Reply:
Got it :+1: